pkg/utils/cidr.go

```diff
@@ -20,8 +20,14 @@ import (
 
 // IPToCidr converts from an ip to a full mask cidr
 func IPToCidr(ip string) string {
-	// TODO: support IPv6
-	return ip + "/32"
+	// TODO: support IPv6 correctly
```

Comment:
do you mean the support of IPV6 is not correctly now? 😃

Reply:
I don't really consider merely appending a "/32" or "/128" to an IP address a correct to convert IP to CIDR. As I know a CIDR more likely refer to a *Range* while IP is to a *Dot*.

Reply:
ok, we can fix it later